lib/jquery-minicolors-rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module JqueryMinicolorsRails
-  VERSION = '2.2.3.1'
+  VERSION = '2.2.6'
```

Comment:
@kostia Does this need a `.1` on the end? 

https://github.com/kostia/jquery-minicolors-rails#versioning

Reply:
Yeah, feels better if the version history looks consistent:
<img width="292" alt="bildschirmfoto 2017-09-22 um 17 38 45" src="https://user-images.githubusercontent.com/9946/30752570-f8e4a23a-9fbc-11e7-8251-32c62ca5ecfe.png">